<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChiaTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabFolders">
       <attribute name="title">
        <string>硬盘</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FoldersWidget" name="tabFoldersWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPlot">
       <attribute name="title">
        <string>P图任务</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="PlotWidget" name="tabPlotWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHPoolMine">
       <attribute name="title">
        <string>HPool矿池挖矿</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="HPoolMineWidget" name="tabHPoolMineWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHuobiPoolMine">
       <attribute name="title">
        <string>火币矿池挖矿</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="HuobiPoolMineWidget" name="tabHuobiPoolMineWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAbout">
       <attribute name="title">
        <string>关于</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="AboutWidget" name="tabAboutWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoldersWidget</class>
   <extends>QWidget</extends>
   <header>widgets.FoldersWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>widgets.PlotWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HPoolMineWidget</class>
   <extends>QWidget</extends>
   <header>widgets.HPoolMineWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HuobiPoolMineWidget</class>
   <extends>QWidget</extends>
   <header>widgets.HuobiPoolMineWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutWidget</class>
   <extends>QWidget</extends>
   <header>widgets.AboutWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
